docs/source/development/01_guidelines.md

```diff
@@ -24,10 +24,12 @@ The `import_*` directives allow to load content statically. It's usually
 small performances improvements. They fully support `tags` so that we may
 get a more convenient way to filter tasks in the play.
 
-#### What to use then?
+~~~{admonition} What to use then?
+:class: tip
```

Comment:
this one needs a check it renders the text :class:

https://github.com/openstack-k8s-operators/ci-framework/blob/825a786be92936dec286d6990c5f5bad85f0f77a/docs/source/development/01_guidelines.md

Reply:
that won't work in the actual markdown apparently.

Reply:
I'm using that specific syntax to get an actual "header" in the doc - but it's not "markdown native" unfortunately. Not the first time I would see a doc source not rendering perfectly when it's built and pushed to read-the-docs or other doc hosting services, to be fair...
But the benefits of this syntax are, imho, worth that little issue with plain markdown output. WDYT?

Reply:
I see, so it will eventually display the right thing when combined into a doc? what is the format of the final doc though is it something other than md? 
ok, i mean, if you're aware of this and want to proceed then fair enough.

Reply:
can you not use ~~~{tip}
is that the same as ~~~{admonition} with the :class: tip ?

the tip one seems to render OK at https://github.com/openstack-k8s-operators/ci-framework/blob/825a786be92936dec286d6990c5f5bad85f0f77a/docs/source/cookbooks/zuul-job-nodeset.md